openmetadata-spec/src/main/resources/json/schema/entity/events/webhook.json

```diff
@@ -35,6 +35,21 @@
       "description": "Secret set by the webhook client used for computing HMAC SHA256 signature of webhook payload and sent in `X-OM-Signature` header in POST requests to publish the events.",
       "type": "string"
     },
+    "headers": {
+      "description": "Custom headers to be sent with the webhook request.",
+      "type": "object",
+      "existingJavaType": "java.util.Map<String, String>"
+    },
+    "json": {
```

Comment:
any reason to add  json here @Siddhanttimeline ?

Reply:
let's remove that

Reply:
Noted.